free-programming-books.md

```diff
@@ -1443,11 +1443,11 @@ For resources on Angular, Backbone, D3, Dojo, Ember, Express, jQuery, Knockout,
 
 * [Introducing Julia wikibook](https://en.wikibooks.org/wiki/Introducing_Julia)
 * [Julia by Example](http://samuelcolvin.github.io/JuliaByExample) - Samuel Colvin (GitHub repo)
+* [Julia language: a concise tutorial](https://www.gitbook.com/book/sylvaticus/julia-language-a-concise-tutorial) - Antonello Lobianco (GitBook - PDF)
```

Comment:
Great contribution. but you're not linking directly to pdf, so no notation is needed. Also, EPUB, MOBI, and HTML versions are also available